scripts/r.mask/r.mask.html

```diff
@@ -1,68 +1,78 @@
+The user should be aware that a mask remains in place until a user removes it
+or renames it. To remove a mask and restore
 raster operations to normal (i.e., all cells of the current region), remove the
-MASK by setting the <b>-r</b> remove MASK flag (<code>r.mask -r</code>).
+mask by setting the <b>-r</b> flag (<code>r.mask -r</code>).
 Alternatively, a mask can be removed using <em>g.remove</em> or by renaming it
 to any other name with <em>g.rename</em>.
+<p>
+The <code>GRASS_MASK</code> environment variable can be used to specify
+the raster map which will be used as a mask. If the environment variable is
+not defined, the name <code>MASK</code> is used instead.
 
 <h2>NOTES</h2>
 
 The above method for specifying a "mask" may seem counterintuitive. Areas
-inside the MASK are not hidden; areas outside the MASK will be ignored until
-the MASK file is removed.
+inside the mask are not hidden; areas outside the mask will be ignored until
+the mask is removed.
 <p>
 <em>r.mask</em> uses <em>r.reclass</em> to create a reclassification of an
-existing raster map and name it <code>MASK</code>. A reclass map takes up less
+existing raster map and names it <code>MASK</code> by default.
+A reclass map takes up less
 space, but is affected by any changes to the underlying map from which it was
 created. The user can select category values from the input raster to use in the
-MASK with the <em>maskcats</em> parameter; if <em>r.mask</em> is run from the
+mask with the <em>maskcats</em> parameter; if <em>r.mask</em> is run from the
 command line, the category values listed in <em>maskcats</em> must be quoted
 (see example below). Note that the <em>maskcats</em> can only be used if the
 input map is an integer map.
 
-<h3>Different ways to create a MASK</h3>
+<h3>Different ways to create a mask</h3>
 
-The <em>r.mask</em> function creates a MASK with values 1 and NULL. But note
-that a MASK can also be created using other functions that have a raster as
-output, by naming the output raster 'MASK'. Such layers could have other
+The <em>r.mask</em> function creates a mask with values 1 and NULL. But note
+that a mask can also be created using other functions that have a raster as
+output, by naming the output raster <code>MASK</code> (by default).
+Such layers could have other
 values than 1 and NULL. The user should therefore be aware that grid cells
-in the MASK map containing <code>NULL</code> or <code>0</code> will replace data with
+in the mask map containing <code>NULL</code> or <code>0</code> will replace data with
 NULL, while cells containing other values will allow data to pass through
 unaltered. This means that:
 <p>
 If a binary map with [0,1] values is used as input in <em>r.mask</em>, all
-raster cells with 0 and 1 will be part of the MASK. This is because
+raster cells with 0 and 1 will be part of the mask. This is because
 <em>r.mask</em> converts all non-NULL cells to 1.
```

Comment:
Is this right? It sounds contradictory. In the first part it says that NULL or 0 will become NULL in the mask, while the second part says that 0 won´t become NULL in the mask...

Reply:
Explaining the mask gets messy. Someone even put "it's counterintuitive" comment to the doc.

This sentence sounds wrong. I'll look into that. Maybe it is meant to say area which is masked out.

The masked, masked out terminology is not helping because the "presence" cells in the mask show rather than hide.